api/projects.ts

```diff
@@ -288,3 +289,24 @@ export async function fetchDeployWarnLogs(
     url: `${PROJECTS_LOGS_API_PATH}/logs/projects/${projectName}/deploys/${deployId}/combined/warn`,
   });
 }
+
+export async function migrateApp(
+  accountId: number,
+  appId: number,
+  projectName: string
+) {
+  return http.post(accountId, {
+    url: `${MIGRATIONS_API_PATH}/migrations`,
+    data: {
+      componentId: appId,
+      componentType: 'PUBLIC_APP_ID',
+      projectName,
+    },
+  });
+}
+
+export async function checkMigrationStatus(accountId: number, id: number) {
```

Comment:
Do these api requests have return types that we can include?

Reply:
Addressed in https://github.com/HubSpot/hubspot-local-dev-lib/pull/130/commits/d19b22a17f35d945c8645df41b3e618bc3f8b3f8